src/parsing/include/antioch/ascii_parser.h

```diff
@@ -57,12 +57,9 @@ namespace Antioch
   template <class NumericType>
   class ChemicalMixture;
 
-  template <class ThermoEval,class NumericType>
+  template <class NumericType>
```

Comment:
No, this is not in a previous release, so there's no backward compatibility to support.